src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/BehandlingSteg.kt

```diff
@@ -95,6 +99,7 @@ enum class StegType(
 
     fun hentNesteSteg(): StegType {
         return when (this) {
+            REVURDERING_ÅRSAK -> VILKÅR
```

Comment:
Nå husker jeg ikke regelsettet her, men hva skjer hvis jeg fyller ut vilkårsiden, vedtakssiden, lager brev og går tilbake, sletter årsak, og legger inn på nytt. 

Kan jeg da sende til beslutter? Eller må jeg "lagre vilkår" på nytt og lagre vedtak på nytt? 😱

Reply:
REVURDERING_ÅRSAK endrer ikke nåværende steget 😁 
Innså det igår, sånn at den beholder steget til behandlingen for å ikke havne i den situasjonen 
https://github.com/navikt/familie-ef-sak/pull/1890/commits/3858dbd9a8936d33794b510caf3986944939ba9a#diff-c3da1e16f220f09ae7f704e983ed873d45a405f240322dc06cf713f245d40f5cR53